hoot-core-test/src/test/cpp/hoot/core/algorithms/AlphaShapeTest.cpp

```diff
@@ -61,6 +63,11 @@ class AlphaShapeTest : public CppUnit::TestFixture
 
 public:
 
+  void setUp()
+  {
+    TestUtils::mkpath("test-output/algorithms");
+  }
+
```

Comment:
A lot of the changes in this PR consist of moving calls to `QDir().mkpath()` up into a `setUp` function that is called before each test in the CPPUNIT framework.